oshi-core/src/main/java/oshi/software/common/AbstractOperatingSystem.java

```diff
@@ -89,6 +89,12 @@ public String getManufacturer() {
         return this.manufacturer;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public OSService[] getServices() {
```

Comment:
Missing the import statement for `OSService` so this is failing compilation.  Same thing for all the platform implementations of this.